docs/source/examples/sentiment.rst

```diff
@@ -0,0 +1,359 @@
+
+Sentiment analysis is one of the major tasks of Natural Language Processing (NLP),
+which identifies writers’ sentiments in sentences. When expressing sentiment,
+we basically uses labels whether it is positive or negative. For example,
+in the case of the dataset used this time, emotions are expressed in 5 classes
+like 1(very negative), 2(negative), 3(neutral), 4(positive), 5(very positive).
+
+.. figure:: ../../image/sentiment/sentiment-treebank.png
+    :scale: 100%
+
+    cited from [1]
+
+Sentiment analysis is implemented with Recursive Neural Networks. Recursive Neural
+Networks are recursive neural nets with a tree structure. NLP often expresses
+sentences in a tree structure, Recursive Neural Networks are often used in NLP.
+First, we explain the training method of Recursive Neural Networks without
+mini-batch processing. After that, as an advanced story, we explain the training
+method of a mini-batched Recursive Neural Network.
+
+In this tutorial, we will learn the following:
+
+#. What is a Recursive Neural Network?
+#. Implementation of sentiment analysis by Recursive Neural Networks using Chainer
+
+    * Training method of a Recursive Neural Network without mini-batches
+    * Training method of a mini-batched Recursive Neural Network
+
+1. What is a Recursive Neural Network? [2]
+==========================================
+
+A Recursive Neural Network is a Recurrent Neural Networks that extends to
+a tree structure. As both networks are often written as RNN, we need to be
+careful which one we are referring to. RNN usually refers to the Recurrent
+Neural Networks, but in natural language processing it sometimes
+refers to Recursive Neural Networks.
+
+Recursive Neural Networks use a tree structure with a fixed number of branches.
+In the case of a binary tree, the hidden state vector of the current node is
+computed from the hidden state vectors of the left and right child nodes,
+as follows:
+
+.. math::
+    {\bf h}_P = a \left( {\bf W} 
+    \left[ \begin{array}{l}
+        {\bf h}_L \\
+        {\bf h}_R \\
+    \end{array} \right]
+    + {\bf b} \right)
+
+.. image:: ../../image/sentiment/rnn.png
+
+This operation is sequentially calculated from the leaf nodes toward the root node.
+Recursive Neural Networks are expected to express relationships between long-distance
+elements compared to Recurrent Neural Networks, because the depth is enough with
+:math:`log_2(T)` if the element count is :math:`T`.
+
+2. Implementation of sentiment analysis by Recursive Neural Networks
+=====================================================================
+
+As shown below, we explain the implementation with Colaboratory.
+You can immediately run the tutorial with a GPU environment in a browser.
+So, please try it!
+
+`chainer-community/chainer-colab-notebook/OfficialExample(ja)/sentiment_ja.ipynb <https://colab.research.google.com/drive/1NQOxHw-JINkDbY4JDT5Tgy8fOuEcbJbv>`_
```

Comment:
Please link to the official example code in the repository for now.
(Let's consider separately about how to introduce Colab links for examples.)

Reply:
Okay!